Wikipedia/Categories/NSAttributedString+WMFHTMLForSite.m

```diff
@@ -11,13 +11,17 @@
 #import "MWKSite.h"
 #import "NSAttributedString+WMFModify.h"
 #import "NSAttributedString+WMFTrim.h"
+#import "UIFont+WMFStyle.h"
 
 @implementation NSAttributedString (WMFHTMLForSite)
 
 + (NSDictionary*)wmf_defaultHTMLOptionsForSite:(MWKSite*)site {
-    UIFont* defaultFont                = [UIFont preferredFontForTextStyle:UIFontTextStyleBody];
-    DTCSSStylesheet* defaultStyleSheet =
-        [[DTCSSStylesheet alloc] initWithStyleBlock:@"img { display: none } "];
+    UIFont* defaultFont = [UIFont wmf_htmlBodyFont];
+    static DTCSSStylesheet* defaultStyleSheet;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        defaultStyleSheet = [[DTCSSStylesheet alloc] initWithStyleBlock:@"img { display: none } "];
```

Comment:
Would be nice if you moved this to a category in the same way you moved the `wmf_htmlBodyFont` to a category.

it's nice for encapsulation, and, honestly, I don't know what this does… so it would be nice to have it in a well named method that conveys it's purpose.